src/jobsautoreport/config.py

```diff
@@ -5,3 +5,4 @@
 ES_PASSWORD = os.environ["ES_PASSWORD"]
 ES_JOB_INDEX = os.environ["ES_JOB_INDEX"]
 SLACK_WEBHOOK_URL = os.environ["SLACK_WEBHOOK_URL"]
+LOG_LEVEL = os.getenv("LOG_LEVEL", "INFO")
```

Comment:
```suggestion
LOG_LEVEL = os.getenv("JOBS_AUTO_REPORT_LOG_LEVEL", "INFO")
```
As defined here: https://github.com/openshift-assisted/prow-jobs-scraper/blob/main/openshift/template.yaml#L130

We should also adapt https://github.com/openshift-assisted/prow-jobs-scraper/blob/main/src/prowjobsscraper/config.py since the variable was renamed

Reply:
Nevermind, Monday morning :man_facepalming: